README.md

```diff
@@ -14,14 +14,14 @@ This specification serves two purposes:
 provides a mechanism to describe a profile in an unambiguous way.
 This mechanism can be used to define additional profiles.
 
-- In addition it defines a **Core Profile** of the Thing Description 
-for use with selected protocols. The <a>Core Profile</a> formalizes
+- In addition it defines a **Base Profile** of the Thing Description 
```

Comment:
There is no "Base Profile". There is an "HTTP Baseline Profile" which may be renamed to "HTTP Basic Profile" (#265), but it only uses HTTP (not "selected profiles" as written here).